ai-platform-unified/notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb

```diff
@@ -1,1092 +1,1335 @@
 {
```

Comment:
<p>same as the comment in the 'control flow' notebook -- this doesn't go out to Vertex AI training/serving, so probably best to pare down this list, and the code below, to just what was created, so as not to confuse the users.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>